### ISSUE TITLE
OWLFile Parser

### ISSUE THREAD

**Issue by Shriyanshmit** [bpellens/owldotnetapi#2]
Hi,
Thanks for your valuable response.
I have some questions related to owl file. 

Ques.1
Suppose I am using an owl file for a device like temperature(send the temperature after a specific time.)
My temperature device have some command(shut down device) and queries(get temperature). How I can declare these command queries in owl file.

Ques-2. Is owl file is useful for find out the functions,properties of a device.

Ques.3-  I go through the test4 function but I having some confusion.
I am using camera.owl(http://protege.cim3.net/file/pub/ontologies/camera/camera.owl) file which contains some properties, function, classes etc.
 I am using online visualisation for owl :(http://owlgred.lumii.lv/online_visualization/sg13)
When I browse my camera.owl file to above link it shows the diagram which shows the relationship between class,objects, properties, functions etc.
Now I am having one question how I will find out the all properties which are link to a class(like we parse XML file and load to c# data set which contains the relationship of each and everything of XML file).
I am having no issue to find out the class node from owlapidotnet Library test4 function but I also want to find out all the link properties, functions, objects, sub classes etc

Ques-4. What is domain range in data property please share any link or helpful solution.
How's I will differentiate between function and properties etc.

Thanks
Shreyansh Jain

**Comment by bpellens** [comment]
Hello,

My apologies for the late answer.

Question 1: You will probably have to create a concept/class for each of
your concepts. So a class for "Command" and a class for "Query", next, you
will have properties such as "hasCommand" and "hasQuery". These properties
can then be added to your device. The range of the properties will be your
Command and Query classes.

Question 2: Yes, in my opinion, you could use something like this.

Question 3: If I understand correctly, this is the same question as the
question you posed earlier. You will have to go to your node and then list
all the properties such as "hasCommand", "hasQuery" and so on. You can do
this by iterating over all the properties in your ontology that have the
class node as the Domain and then searching for these properties within
your node.

Question 4: Could you rephrase this question since I do not understand it.

Kind Regards,

Bram
